Core/HLE/sceNetAdhoc.cpp

```diff
@@ -126,6 +126,9 @@ static int sceNetAdhocPdpCreate(const char* mac, int port, int bufferSize, u32 f
 static int sceNetAdhocPdpSend(int id, const char* mac, u32 port, void* data, int len, int timeout, int flag);
 static int sceNetAdhocPdpRecv(int id, void* addr, void* port, void* buf, void* dataLength, u32 timeout, int flag);
 
+bool __NetAdhocConnected() {
+	return netAdhocInited && netAdhocctlInited && adhocctlState == ADHOCCTL_STATE_CONNECTED;
```

Comment:
it should be `(adhocctlState == ADHOCCTL_STATE_CONNECTED) || (adhocctlState == ADHOCCTL_STATE_GAMEMODE)` as gamemode is also in connected state, similar to https://github.com/hrydgard/ppsspp/blob/d2ebd36ab963bdf21f8fae50c6cc5eec21cc2426/Core/HLE/sceNetAdhoc.cpp#L2285-L2287

Reply:
Fixed it up, please test :)

Reply:
I only tested this on 2 games, Naruto Shippuden Ultimate Ninja Heroes 3 (normal adhoc), and Bomberman (gamemode), seems to be okay.

Reply:
Thanks for testing! I think it should be generally fine, I guess we'll get bug reports if it isn't.